tests/kernel_performance/install_qatestset.pm

```diff
@@ -27,6 +27,10 @@ sub install_pkg {
 sub setup_environment {
     my $runid             = get_required_var('QASET_RUNID');
     my $mitigation_switch = get_required_var('MITIGATION_SWITCH');
+    my $ver_cfg           = get_var("VER_CFG");
```

Comment:
Does it make sense to use `get_required_var` here?

Reply:
updated,thanks for reviewing